contracts/Core/VoteManager.sol

```diff
@@ -110,6 +117,12 @@ contract VoteManager is Initializable, ACL, VoteStorage {
         }
     }
 
+    function isAssetAllotedToStaker(uint256 stakerId, uint256 iteration, uint256 assetId) public view initialized returns (bool)
```

Comment:
Not sure if the logic is correct. let's clarify on call.

Reply:
sure